docs/ide4c/vscode/launch.json

```diff
@@ -0,0 +1,66 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "intltest Launch", 
+            "type": "cppdbg",
+            "request": "launch",
+            "targetArchitecture": "x64",
+            "program": "${workspaceFolder}/test/intltest/intltest",
+            "environment": [
+                {
+                    "name": "LD_LIBRARY_PATH", // Use "DYLD_LIBRARY_PATH" for mac osx.
```

Comment:
The red highlighting here of // comments indicates that they are not supported in .json, right? Does VS Code handle them, or does one have to strip them while copying into the .vscode folder?

Reply:
Vscode ignores // comments.

Reply:
Yes, vscode ignores them. and I think those comments are pretty useful when in this place.